lib/cosmos/utilities/store.rb

```diff
@@ -44,49 +45,25 @@ def self.instance(pool_size = 10)
       end
     end
 
-    def build_redis
-      return Redis.new(url: @redis_url)
+    # Delegate all unknown class methods to delegate to the instance
+    ruby2_keywords def self.method_missing(message, *args, &block)
+      self.instance.send(message, *args, &block)
+    end
+
+    # Delegate all unknown methods to redis through the @redis_pool
+    ruby2_keywords def method_missing(message, *args, &block)
+      @redis_pool.with { |redis| redis.send(message, *args, &block) }
```

Comment:
The magic of method_missing! Discovered I needed to add the ruby2_keywords to make it work across the board.

Reply:
Nice!